src/league.ts

```diff
@@ -506,17 +506,19 @@ export class League {
     getTeamByPlayerName(playerName: string) {
         playerName = playerName.toLowerCase()
         const directTeamMapping =
-            this._playerLookup[playerName.toLowerCase()] &&
-            this._playerLookup[playerName.toLowerCase()].team
+            this._playerLookup[playerName] &&
+            this._playerLookup[playerName].team
         if (directTeamMapping) return directTeamMapping
         // Try to find the team by looking through the pairings for this
         // playername.  This will find alternates.
-        const teams: Team[] = this._teams.filter((t) => {
-            const playerIds = t.players.map((p) => p.username.toLowerCase())
-            return playerIds.filter((n) => n === playerName).length > 0
-        })
-        if (teams.length > 0) {
-            return teams[0]
+        const possiblePairings = this.findPairing(playerName)
```

Comment:
One thing I noticed - findPairing compares usernames using .includes() instead of lodash.isEqual(), feels like that should probably be changed?

Reply:
Yeah, we should be comparing for equality there. Good catch.

Reply:
Although if we do change that, then the signature for that function should probably be `Pairing | undefined`

Reply:
Let's open a new issue for this one and get the rest of this live once you update the checks.